helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/secret-elasticsearch.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if not .Values.global.elasticsearch.existingSecretName }}
```

Comment:
Why is the `existingSecret` only provided through the global scope? We usually use global as a fallback for the root scope in case the values isn't available (and as a side effect provides convenient way of spreading the values)

Reply:
took a shortcut and forgot to fix it later, thanks for pointing it